### PR TITLE
Adicionando Lista de Aplicações em Electron

### PR DESCRIPTION
**Pull request by pbaesse**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/portuguese/electron/index.md [hunk 1]
```diff
@@ -23,8 +23,11 @@ Alguns aplicativos criados usando o Electron incluem:
 
 ### Referências adicionais de informação
 
+https://electronjs.org/apps
+
 *   [Site oficial](https://electronjs.org/)
+*   [Lista de Aplicações em Electron](https://electronjs.org/apps)
```
Comment (fabiotp91) [inline guide/portuguese/electron/index.md]: This is redundant with the main website just over it

**Review by fabiotp91 — DISMISSED** [review]
Please remove the List of applications link because it's just redundancy with the main website link just over it.

**Comment by fabiotp91** [comment]
> 
> 
>     * [ ]  I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
> 
>     * [ ]  My pull request has a descriptive title (not a vague title like `Update index.md`)
> 
>     * [ ]  My pull request targets the `master` branch of freeCodeCamp.
> 
>     * [ ]  None of my changes are plagiarized from another source without proper attribution.
> 
>     * [ ]  My article does not contain shortened URLs or affiliate links.
> 
> 
> If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.
> 
> Closes #XXXXX

And please check the checkboxes

File: guide/portuguese/electron/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/portuguese/electron/index.md]: Are you suggesting to remove line 26 and 29?

Comment (fabiotp91) [reply guide/portuguese/electron/index.md]: Only line 29

Comment (RandellDawson) [reply guide/portuguese/electron/index.md]: I will remove both of them.

**Comment by RandellDawson** [comment]
Congrats @pbaesse  on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**